site/ar/tutorials/images/cnn.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>Add &lt;div dir="rtl"&gt;&lt;/div&gt; around the text block so it renders correctly. Otherwise, the current words order doesn't have a meaning. </p><p> </p><pre class="ql-syntax" spellcheck="false">&lt;div dir="rtl"&gt;
في هذا الدرس سيتم شرح تدريب بسيط لشبكة عصبية Convolutional Neural Network لتصنيف CIFAR images. Keras Sequential API, هذا الدرس يستخدم Keras Sequential API لنتمكن من تدريب الشبكة العصبية في بضع سطور . 
&lt;/div&gt;
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs-l10n/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>